mdx/api/index.mdx

```diff
@@ -0,0 +1,776 @@
+| exchangeAddress       | The address of the 0x exchange smart contract where the transaction should be sent to and that will settle the order.                                                                                                                                                                                                                                                              |
+| chainId               | The ID of the Ethereum chain where the `exchangeAddress` is located.                                                                                                                                                                                                                                                                   |
+| signature             | The signature of the fields above using the private key of `makerAddress`.                                                                                                                                                                                                                                                             |
+
+## Misc.
+
+*   All requests and responses should be of "application/json" content type
+*   All token amounts are sent in amounts of the smallest level of precision (base units). (e.g if a token has 18 decimal places, selling 1 token would show up as selling "1000000000000000000" units by this API).
+*   All addresses are sent as lower-case (non-checksummed) Ethereum addresses with the `0x` prefix.
+*   All parameters should use **lowerCamelCase**.
+
+# Swap
+
+Description of /swap endpoint.
+
+## GET /swap/quote
+
+Get a quote.
+
+### Request 
+
+| Query Param        | Description                                                                                                                                                                                                         |
+| ------------       | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| sellToken          | The ERC20 token address or symbol of the token you want to send. "ETH" can be provided as a valid `sellToken`.                                                                                                      |
+| buyToken           | The ERC20 token address or symbol of the token you want to receive.                                                                                                                                                 |
+| sellAmount         | (Optional) The amount of `sellToken` (in `sellToken` units) you want to send. If `sellAmount` is provided, `buyAmount` will be ignored.                                                                                           |
```

Comment:
Got bit here -- either sellAmount or buyAmount is required (even though both say optional)